src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -628,6 +631,7 @@ namespace JSONHelpers
                     abort(); // TODO(stefan): Exception safety
                 }
                 customZoneSetsMap[uuid] = zoneSetData;
+                SaveFancyZonesData();
```

Comment:
This one is not needed.. As MigrateCustomZoneSetsFromRegistry is only called from LoadFancyZonesData(). And there is a SaveFancyZonesData call right after it there